apps/test/unit/templates/plugins/externalLinksTest.js

```diff
@@ -0,0 +1,30 @@
+import {assert} from '../../../util/configuredChai';
+import {isExternalLink} from '@cdo/apps/templates/plugins/externalLinks';
+
+describe('external links remark plugin', () => {
+  it('treats relative links as internal', () => {
+    assert.equal(isExternalLink('/educate'), false);
+    assert.equal(isExternalLink('../../educate'), false);
+    assert.equal(isExternalLink('/hoc/1'), false);
+    assert.equal(isExternalLink('puzzle/1'), false);
+    assert.equal(isExternalLink('test'), false);
+  });
+
+  it('treats *.code.org links as internal', () => {
+    assert.equal(isExternalLink('//code.org/educate'), false);
+    assert.equal(isExternalLink('http://code.org/educate'), false);
+    assert.equal(isExternalLink('https://code.org/educate'), false);
+    assert.equal(isExternalLink('https://studio.code.org/'), false);
+    assert.equal(isExternalLink('//localhost-studio.code.org:3000/s/1'), false);
+  });
+
+  it('treats other domain links as internal', () => {
```

Comment:
Is this supposed to be

```suggestion
  it('treats other domain links as external', () => {
```